test/ui_test_label_studio.py

```diff
@@ -0,0 +1,213 @@
+#
+# Author: Hovhannes Atoyan (hovhannes.atoyan@gmail.com)
+# UI automation for Label Studio
+# Automated base steps for Image labeling (Setup, Import, Start, Export)
+#
+
+import os
+import time
+import datetime
+import unittest
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.firefox.firefox_binary import FirefoxBinary
+from selenium.common.exceptions import NoSuchElementException
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.common.action_chains import ActionChains
+
+
+class myTestCase(unittest.TestCase):
+    def setUp(self):
+        # self.driver = webdriver.Firefox()
+        #
+        self.downloadDir = './'
+        fp = webdriver.FirefoxProfile()
+        fp.set_preference("browser.download.folderList", 2)
+        fp.set_preference("browser.download.manager.showWhenStarting", False)
+        fp.set_preference("browser.download.dir", self.downloadDir)
+        # fp.set_preference("browser.helperApps.neverAsk.saveToDisk", "attachment/csv")
+        # fp.set_preference("browser.helperApps.neverAsk.saveToDisk", "attachment/json")
+        fp.set_preference("browser.helperApps.neverAsk.saveToDisk", "attachment/zip")
+        self.driver = webdriver.Firefox(firefox_profile=fp)
+        #
+        self.wait = WebDriverWait(self.driver, 10)
+        self.probel5 = '      '
```

Comment:
?

Reply:
It were for logging and I removed it for now)